client/layout/guided-tours/style.scss

```diff
@@ -121,10 +121,10 @@
 	.button:nth-child(1) {
 		margin-right: 4%;
 	}
-	.button:nth-child(2) {
+	.guided-tours__quit-button {
 		color: darken( $white, 20% );
 		background: none;
-		border: none;
+		border: 1px darken( $white, 20% ) solid;
```

Comment:
Lets use opacity here instead of darkening white, which just makes it a muddy gray.

Reply:
Sorry, opacity is prolly the wrong word. Lets use `rgba()`